bzt/10-base.json

```diff
@@ -192,6 +192,16 @@
           "dump-locations": true
         }
       }
+    },
+    "func": {
```

Comment:
Do you assume that we can have non-functional consolidator and functional executor? Shouldn't it be a global flag under top level "settings"?